model/src/main/kotlin/Scope.kt

```diff
@@ -51,12 +51,15 @@ data class Scope(
      * depth. If [includeErroneous] is true, [PackageReference]s with issues (but not their dependencies without issues)
```

Comment:
Comment statement needs to be updates as `includeErroneous` has been renamed to `filterPredicate`

Reply:
Done.